Candidate: CVE-2014-0480
PublicDate: 2014-08-26
References:
 http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-0480
 https://www.djangoproject.com/weblog/2014/aug/20/security/
Description:
 The core.urlresolvers.reverse function in Django before 1.4.14, 1.5.x
 before 1.5.9, 1.6.x before 1.6.6, and 1.7 before release candidate 3 does
 not properly validate URLs, which allows remote attackers to conduct
 phishing attacks via a // (slash slash) in a URL, which triggers a
 scheme-relative URL to be generated.
Ubuntu-Description:
Notes:
Bugs:
Priority: low
Discovered-by:
Assigned-to: sarnold

Patches_python-django:
 vendor: https://www.debian.org/security/2014/dsa-3010
upstream_python-django: released (1.6.6-1)
lucid_python-django: needs-triage
precise_python-django: needs-triage
trusty_python-django: needed
devel_python-django: needed
